Automattic/wp-calypso
# Domain Transfers: Congrats page details

Related to https://github.com/Automattic/dotcom-forge/issues/3903
Follow up from https://github.com/Automattic/wp-calypso/pull/82237

## Proposed Changes

Update CSS details to match Figma and confirm that CTAs are tracked.

Figma: VLoPHWqcewxNKZYjjkDu3O-fi-2996%3A3335

| Before | After |
| --- | --- |
| ![image](https://github.com/Automattic/wp-calypso/assets/402286/f5838192-a9c6-40e5-904b-0b5a539c3815) | ![image](https://github.com/Automattic/wp-calypso/assets/402286/3cddb5a2-0ba3-4ea7-9b66-c37b9eada716) |

## Testing Instructions

* Go to `http://calypso.localhost:3000/checkout/domain-transfer-to-any-user/thank-you/your-domain`
* Check the design details
* Check track events for all CTA

## Pre-merge Checklist
